foundations/html_css/flexbox/flexbox-growing-and-shrinking.md

```diff
@@ -66,6 +66,21 @@ In practice you will likely not be using complex values for `flex-grow`, `flex-s
 
 It _is_ possible to get fancy, and set up layouts where some columns relate to each other in a specific ratio, so it's useful to know that you can use other values, but those are relatively rare.
 
+### Assignment
+
+<div class="lesson-content__panel" markdown="1">
+
+1. Read <a href ="https://www.w3.org/TR/css-flexbox-1/#flex-common">this section</a> to understand the basic values of common flex shorthand values
```

Comment:
```suggestion
1. Read <a href ="https://www.w3.org/TR/css-flexbox-1/#flex-common">W3C's flex section</a> to understand the basic values of common flex shorthand values
```
**suggestion:** This will help improve the link readability by making it more descriptive. See here: https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#use-informative-markdown-link-titles